TheThingsNetwork/lorawan-stack
# Update and fix storybook

#### Summary



This PR makes storybook work again!

#### Changes



- Update storybook and dependencies
- Add docs addon
- Adapt stories to new format

#### Testing



1. Build the front end with `tools/bin/mage js:clean js:cleanDeps jsSdk:clean jsSdk:cleanDeps js:build`
2. Run `yarn sb dev --config-dir ./config/storybook -p 9001` to start storybook
3. See components and their docs


#### Notes for Reviewers



...

#### Checklist
